### PR TITLE
#2984 Drawing app using Js

### LINKED ISSUE #2984: Drawing App using Js
**Describe the project you want to add with tech stack**
I would like to add a drawing app project using JavaScript. The project will include the following features:

- Three shapes (circle, square, and triangle) that users can choose from
- Option to select between solid filled figures and normal figures for the chosen shape
- Brush tool for drawing, with adjustable thickness up to 100%
- Eraser tool for erasing drawn elements, also adjustable up to 100%
- Five color options: white, black, red, green, and blue
- Color palette to provide shades of the selected color upon hovering

**Expected behavior**
The drawing app should allow users to choose a shape and its fill type, and then draw or erase using the selected brush or eraser tool. The thickness of the brush and eraser should be adjustable up to 100%. Users should be able to select one of the five available colors and see shades of the selected color when hovering over the color palette.

### PR DESCRIPTION
**Pull request by Yashika-Agrawal**
# Title and Issue number 
#2984 Drawing app using Js #2984

Close #2984 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

**Comment by Yashika-Agrawal** [comment]
@MohitGupta121 , hey the label is not added , please add the gssoc and level number here.

**Comment by MohitGupta121** [comment]
@Yashika-Agrawal this Pr is closed not merged.

**Comment by Yashika-Agrawal** [comment]
@MohitGupta121 but why?

File: Js-Projects/Drawing-app/drawing-app-screenshot.jpeg
Comment (MohitGupta121) [inline Js-Projects/Drawing-app/drawing-app-screenshot.jpeg:1]: Remove the image from PR and upload it somewhere and then use it link in your project to load image.

**Comment by Yashika-Agrawal** [comment]
I added the image in images folder from there i have linked it in readme.

**Comment by MohitGupta121** [comment]
> I added the image in images folder from there i have linked it in readme.

remove that image folder and fetch images from somewhere through link, github provides an upload image use that and you got link to that image.

**Review by MohitGupta121 — APPROVED** [review]

**Pull request merged**